src/components/body/pillars/Pillars.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import ReactCSSTransitionReplace from 'react-css-transition-replace';
+import '../../../App.css';
```

Comment:
You shouldn't need to re-import the app.css. If need be, please make your own Pillars.css file

Reply:
Fixed!